schemas/put-artifact-response.yml

```diff
@@ -0,0 +1,13 @@
+$schema:            http://json-schema.org/draft-04/schema#
+title:              "Put Artifact Response"
+description: complete a blob artifact upload
+type:               object
+properties:
+  outcome:
+    type: string
+    enum:
+      - success
+      - failure
```

Comment:
Give a request conflict or some http error if it fails... If you fail to upload then:
A) you gave the wrong etags (400 error)
B) You did upload the parts first (400 error)
C) Something went wrong with the request (500 error)

Client implementors are not likely to handle this error. Note: IMO this doesn't have to provide any response schema, we don't have to provide a response other than 200 OK, or 4xx/5xx...